src/App.tsx

```diff
@@ -17,18 +38,38 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                selectedStatus={status}
+                selectStatus={setStatus}
+                selectQuery={setQuery}
+                selectedQuery={query}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading
+                ? (<Loader />)
+                : (
+                  <TodoList
+                    selectedQuery={query}
+                    selectedStatus={status}
+                    todos={todos}
+                    selectedTodoId={todoId}
+                    selectTodo={setTodoId}
+                  />
+                )}
+
             </div>
           </div>
         </div>
       </div>
-
-      <TodoModal />
+      {todoId && (
```

Comment:
```suggestion
      {!!todoId && (
```
<img width="434" alt="Screenshot 2022-09-29 at 23 16 02" src="https://user-images.githubusercontent.com/47606379/193133192-8228dfd9-ad59-4ce6-b006-4039336a16bd.png">